### PR TITLE
Add vpd tooling to binaries

### PR DESCRIPTION
**Pull request by rctl**
Add vpd tooling to list of binaries

Signed-off-by: Rasmus Jönsson <rctl@fb.com>

### CODE REVIEW

**Comment by rctl** [comment]
cc @insomniacslk

**Comment by rctl** [comment]
Uncompressed 40K and compressed around 16K, probably smaller when we compress together with other stuff

**Comment by insomniacslk** [comment]
you should also add the new binary here, https://github.com/systemboot/binaries/blob/master/.travis.yml#L21

**Comment by insomniacslk** [comment]
the build is failing on kexec, probably something has changed in the default compiler flags on travis-ci, I'll check that

**Comment by insomniacslk** [comment]
Fixed in https://github.com/systemboot/binaries/pull/10, can you please rebase?

File: build.sh [hunk 1]
```diff
@@ -41,3 +42,13 @@ cd "memtester-${MEMTESTER_VERSION}"
 make # build statically
 du -hs memtester
 ldd memtester
+
+# build vpd
+cd "${OUTDIR}"
+git clone https://chromium.googlesource.com/chromiumos/platform/vpd
+cd vpd
+git checkout "${VPD_VERSION}"
+make
```
Comment (insomniacslk) [inline build.sh:55]: after #10 can you add `CC=musl-gcc` here?

**Comment by rctl** [comment]
@insomniacslk amended with rebase and suggested additional flag for make :) Thanks for fixing travis in #10

**Review by insomniacslk — APPROVED** [review]
looks good, thanks!

**Pull request merged**